### PR TITLE
[v7r3] HTTPS services for python3

### PR DESCRIPTION
**Pull request by chaen**
BEGINRELEASENOTES
Thank you for writing the text to appear in the release notes. It will show up
exactly as it appears between the two bold lines

Please follow the template:
*Subsystem
NEW/CHANGE/FIX: explanation

For examples look into release.notes

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/DataManagementSystem/Service/TornadoFileCatalogHandler.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
 import os
 
 from io import BytesIO
```
Comment (fstagni) [inline src/DIRAC/DataManagementSystem/Service/TornadoFileCatalogHandler.py:24]: This import is now useless, I think

File: src/DIRAC/DataManagementSystem/Service/TornadoFileCatalogHandler.py [hunk 2]
```diff
@@ -22,6 +22,7 @@
...
+from six import StringIO
 # from DIRAC
 from DIRAC.Core.DISET.RequestHandler import getServiceOption
```
Comment (fstagni) [inline src/DIRAC/DataManagementSystem/Service/TornadoFileCatalogHandler.py:27]: This, and other imports in this module don't seem to be used.

File: src/DIRAC/DataManagementSystem/Service/TornadoFileCatalogHandler.py [hunk 3]
```diff
@@ -60,7 +61,8 @@ def export_streamToClient(self, seName):
     retVal = self.getSEDump(seName)
 
     try:
-      csvOutput = BytesIO()
+      # csvOutput = BytesIO()
```
Comment (fstagni) [inline src/DIRAC/DataManagementSystem/Service/TornadoFileCatalogHandler.py:64]: remove?

File: tests/CI/install_server.sh [hunk 4]
```diff
@@ -46,6 +54,8 @@ X509_CERT_DIR=$SERVERINSTALLDIR/etc/grid-security/certificates/ fullInstallDIRAC
 
 echo -e "*** $(date -u) **** Server INSTALLATION DONE ****\n"
 
+pip install git+https://github.com/chaen/tornado_m2crypto.git@sslProxy
```
Comment (fstagni) [inline tests/CI/install_server.sh:57]: Maybe we first merge the tornado_m2crypto PR?

**Review by fstagni — COMMENTED** [review]
You committed by mistake src/DIRAC/DataManagementSystem/DB/MultiVOFileCatalogWithFkAndPsDB.sql

**Comment by chaen** [comment]
Argh, this is a side effect of running `integration_tests.py`.  I am not even sure this file makes any sense at all... anyway, I'll fix it 

All your comments are correct, It's just all in testing phase :-)

File: src/DIRAC/DataManagementSystem/Service/TornadoFileCatalogHandler.py (hunk 1 above)
Comment (chaen) [reply src/DIRAC/DataManagementSystem/Service/TornadoFileCatalogHandler.py:24]: Yep, removed

File: src/DIRAC/DataManagementSystem/Service/TornadoFileCatalogHandler.py (hunk 2 above)
Comment (chaen) [reply src/DIRAC/DataManagementSystem/Service/TornadoFileCatalogHandler.py:27]: I've cleaned them

**Review by fstagni — APPROVED** [review]

**Pull request merged**